fink_science/image_classification/README.md

```diff
@@ -0,0 +1,39 @@
+# Image Classification
+
+This module perform a classification of alerts based on image visual content. It use only the cutoutScience image and return a single string label which describe image alert. 
```

Comment:
Typos: This module **performs** a classification of alerts based on image visual content. It **uses** only the cutoutScience image and **returns** a single string label which **describes** image alert **class**